kura/org.eclipse.kura.core/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package: javax.crypto,
  javax.net,
  javax.net.ssl,
  javax.sql,
+ org.apache.commons.io;version="[2.4,3)",
```

Comment:
In Kura, the .0 is also mentioned everywhere as it is more visible at first glance. I suggest you to add 3.0.